From a81c294d2a71affbcb1e114070369c632606ceeb Mon Sep 17 00:00:00 2001
From: Joseph Liu <kwliu@nuvoton.com>
Date: Thu, 4 Jun 2026 15:20:04 +0800
Subject: [PATCH] Add more PSU devices support

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Joseph Liu <kwliu@nuvoton.com>
---
 src/psu/PSUSensorMain.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/psu/PSUSensorMain.cpp b/src/psu/PSUSensorMain.cpp
index 93df00ee8..706f5c056 100644
--- a/src/psu/PSUSensorMain.cpp
+++ b/src/psu/PSUSensorMain.cpp
@@ -157,6 +157,8 @@ static constexpr auto sensorTypes = std::to_array<
     {"XDPE11280", I2CDeviceType{"xdpe11280", true}},
     {"XDPE12284", I2CDeviceType{"xdpe12284", true}},
     {"XDPE152C4", I2CDeviceType{"xdpe152c4", true}},
+    {"TPS53659", I2CDeviceType{"tps53659", true}},
+    {"TPS53679", I2CDeviceType{"tps53679", true}},
 });
 
 enum class DevTypes
-- 
2.43.0

